openwisp-config/files/openwisp.agent

```diff
@@ -323,7 +327,6 @@ post_registration_hook() {
 
 # applies a specified configuration archive
 apply_configuration() {
-	local sleep_time=${2-5}
```

Comment:
the default was 5, but you're setting it to zero, I think we should set it to 5.